src/main/resources/i18n/aerospace/common.properties

```diff
@@ -365,4 +365,7 @@ BACK_TO_HOME=Return to home
 BACK_TO_LOGIN_PAGE=Return to login page
 CATEGORIES_EMPTY_CONTENTS_MESSAGE=No categories exist.
 ITEMS_EMPTY_CONTENTS_MESSAGE=No items exist.
-ORDERS_EMPTY_CONTENTS_MESSAGE=No orders exist.
\ No newline at end of file
+ORDERS_EMPTY_CONTENTS_MESSAGE=No orders exist.
+409_ERROR_TITLE=409-Conflict
+ROLE_NOT_MATCH=The user roles in Keycloak does not match the user roles in this application, please return to home.
```

Comment:
Change this to: The user in Keycloak has mismatching roles with user in this application.